include/cocaine/framework/sender.hpp

```diff
@@ -53,11 +53,19 @@ class basic_sender_t {
     template<class Event, class... Args>
     auto
     send(Args&&... args) -> task<void>::future_type {
-        return send(io::encoded<Event>(id, std::forward<Args>(args)...));
+        namespace ph = std::placeholders;
```

Comment:
ok, but what's wrong with it? It's located in function scope, so does not affect outer namespace.